server/src/domain/asset/asset.repository.ts

```diff
@@ -23,6 +23,7 @@ export interface LivePhotoSearchOptions {
 
 export interface MapMarkerSearchOptions {
   isFavorite?: boolean;
+  includeArchived?: boolean;
```

Comment:
The main timeline has this same situation. In that case the endpoint defaults to no value. The clients have to send `false` to exclude them. Can we do the same here? So:
- Rename this to `isArchived`
- Change web/mobile to send `isArchived: false` by default, now configurable in the web.

What do you think about using `isArchived` instead? We can then use it

Reply:
isArchived is already used to toggle between Archived and Not Archived, not the objective of this option!

Reply:
I know, but if you omit it, that can return all assets, and that provide three options/states, correct?:
- isArchived (not set) = all photos (including archived assets)
- isArchived: true = only archived
- isArchived: false = only unarchived

Reply:
I believe in the web you want it to only return assets that are not archived (so send `isArchived: false` by default) and an option to include them (so don't send `isArchived` at all, to include all assets)

Reply:
now it's using the already known isArchived, but imo, this limits future opportunity to "Toggle Between", and makes some confusion on the scope of the property, but if you think thats better i'm ok!

Reply:
Can you help me understand how this limits future toggling exactly? I am not following. What example specifically would not work?

Reply:
for example, if you wanted to give the option to see "only archived", it's a remote possibility, but it would be impossible to manage with a single variable.

Reply:
Why would sending `isArchived: true` not work for that case?

Reply:
nevermind man, drop the PR.....